addons/arsenal/initSettings.sqf

```diff
@@ -33,26 +33,47 @@ private _category = LLSTRING(settingCategory);
     true
 ] call CBA_fnc_addSetting;
 
+[
+    QGVAR(defaultToFavorites),
+    "CHECKBOX",
+    [LSTRING(defaultToFavoritesSetting), LSTRING(defaultToFavoritesTooltip)],
+    _category,
+    false,
+    -2 // never overwrite the client
```

Comment:
See https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#create-a-custom-setting-for-mission-or-mod:

```suggestion
    2 // never overwrite the client
```

Reply:
Derp.